deployments/liqo/templates/liqo-clusterid-configmap.yaml

```diff
@@ -0,0 +1,19 @@
+{{- $clusterIdConfig := (merge (dict "name" "clusterid-configmap" "module" "clusterid-configmap") .) -}}
+{{- $name := include "liqo.prefixedName" $clusterIdConfig -}}
+{{- $oldObject := (lookup "v1" "ConfigMap" .Release.Namespace $name) -}}
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    {{- include "liqo.labels" $clusterIdConfig | nindent 4 }}
+  name: {{ include "liqo.prefixedName" $clusterIdConfig }}
+data:
+  {{- if (not $oldObject) }}
+  CLUSTER_ID: {{ uuidv4 }}
+  {{- else }}
+  CLUSTER_ID: {{ $oldObject.data.CLUSTER_ID }}
+  {{- end }}
+  {{- if .Values.discovery.config.clusterName }}
+  CLUSTER_NAME: {{ .Values.discovery.config.clusterName }}
+  {{- end }}
```

Comment:
It seems to me this is currently not used (don't know if intentionally).
Still, would remove the `if` part, as AFAIK it would "break" the pods if mounted as value and not present

Reply:
This value is not intended to be mounted by any pod. Pods that need the ClusterName values take it directly from `.Values.discovery.config.clusterName` to ensure they restart on name updates.

This value is to be easily readable by a user that does not have to read it from containers arguments.

Another solution can be to completely drop this value from this ConfigMap and set the whole ConfigMap as immutable, I really don't know which is a better solution.

Reply:
Got it. The current solution seems perfectly reasonable to me then.